default/python/universe_generation/empires.py

```diff
@@ -239,6 +238,27 @@ def add_planets_to_vicinity(vicinity, num_planets, gsd):
         num_planets -= 1
 
 
```

Comment:
get_system_within_jumps() has nothing to do with placing the positions in the galaxy.  I would rather not change it in this PR.  I expand on why I think it should be moved below.